src/core/recv_buffer.c

```diff
@@ -419,58 +421,100 @@ QuicRecvBufferCopyIntoChunks(
                 RecvBuffer->Chunks.Flink, // First chunk
                 QUIC_RECV_CHUNK,
                 Link);
-        uint32_t ChunkLength;
+
+        uint64_t ChunkLength;
```

Comment:
Does this really need to be 64 bits? We only use 31 bits for the allocation length of a chunk. So it should be impossible for ChunkLength to be bigger, right?